tests/plugins/balloontoolbar/manual/context-api.md

```diff
@@ -0,0 +1,11 @@
+@bender-ui: collapsed
+@bender-tags: 4.8.0, feature, 1346, balloontoolbar
+@bender-ckeditor-plugins: wysiwygarea,toolbar,undo,basicstyles,notification,floatingspace,sourcearea,list,link
+
+Check if `balloontoolbar` api is available in plugin.init().
+
+This test should not throw error after page load.
+
+Open console and check if `editor.balloonToolbars should be initialized` error has been thrown.
+
+If not, test passed.
```

Comment:
When creating tests description you may try to follow our common pattern for tests scenarios like:

![image](https://user-images.githubusercontent.com/1061942/34672089-637c32d8-f47d-11e7-96db-bb3f303c6ad5.png)